docs/reference/src/main/asciidoc/configure.asciidoc

```diff
@@ -273,6 +273,38 @@ The functionality is implemented as a built-in portable extension processing all
 |`org.jboss.weld.bootstrap.vetoTypesWithoutBeanDefiningAnnotation` || A regular expression. If a non-empty string, then all annotated types whose `AnnotatedType#getJavaClass().getName()` matches this pattern are vetoed if not annotated with a bean defining annotation.
 |=======================================================================
 
+[[remove-unused-beans]]
+==== Memory consumption optimization - removing unused beans
+
+CDI applications consist of user-defined beans implementing the business logic, but also beans coming from libraries (e.g. DeltaSpike) and integrations (e.g. various Java EE technologies - JSF, Bean Validation, Batch).
+Most applications actually use only a small part of the beans provided by libraries and integrations.
+Still, Weld has to retain the metadata for all the beans in memory which can be considerably wast footprint, depending on the application.
```

Comment:
`wast` -> `vast`
sorry that was typo on my side in the first place, but the comment probably slipped you before you changed this.

Reply:
Type after typo ;-)